src/components/v5/common/CompletedAction/partials/ManageReputation/ManageReputation.tsx

```diff
@@ -0,0 +1,166 @@
+import ManageReputationTableInMotion from './partials/ManageReputationTableInMotion/index.ts';
+import { type ManageReputationProps } from './types.ts';
+
+const displayName = 'v5.common.CompletedAction.partials.ManageReputation';
+
+const ManageReputation: FC<ManageReputationProps> = ({ action }) => {
+  const {
+    isMotion,
+    transactionHash,
+    metadata,
+    initiatorUser,
+    recipientUser,
+    amount,
+  } = action;
+  const { networkMotionState } = useGetActionData(transactionHash);
+  const motionFinished =
+    networkMotionState === NetworkMotionState.Finalizable ||
+    networkMotionState === NetworkMotionState.Finalized ||
+    networkMotionState === NetworkMotionState.Failed;
+
+  const isSmite =
+    action.type === ColonyActionType.EmitDomainReputationPenalty ||
+    action.type === ColonyActionType.EmitDomainReputationPenaltyMotion;
+
+  const positiveAmountValue = new Decimal(amount || '0').abs().toString();
+
+  return (
+    <>
+      <ActionTitle>
+        {metadata?.customTitle ||
+          formatText({ id: 'actions.manageReputation' })}
+      </ActionTitle>
+      <ActionSubtitle>
+        {formatText(
+          {
+            id: 'action.title',
+          },
+          {
+            actionType: action.type,
+            initiator: initiatorUser ? (
+              <UserPopover
+                userName={initiatorUser.profile?.displayName}
+                walletAddress={initiatorUser.walletAddress}
+                user={initiatorUser}
+                withVerifiedBadge={false}
+              >
+                {initiatorUser.profile?.displayName}
+              </UserPopover>
+            ) : null,
+            recipient: recipientUser ? (
+              <UserPopover
+                userName={recipientUser.profile?.displayName}
+                walletAddress={recipientUser.walletAddress}
+                user={recipientUser}
+                withVerifiedBadge={false}
+              >
+                {recipientUser.profile?.displayName}
+              </UserPopover>
+            ) : null,
+            reputationChange: amount
+              ? formatReputationChange(amount, DEFAULT_TOKEN_DECIMALS)
+              : '0',
+            reputationChangeNumeral: positiveAmountValue ? (
+              <Numeral
+                value={positiveAmountValue}
+                decimals={DEFAULT_TOKEN_DECIMALS}
+              />
+            ) : (
+              formatText({
+                id: 'actionSidebar.metadataDescription.anAmount',
+              })
+            ),
+          },
+        )}
+      </ActionSubtitle>
+      <ActionDataGrid>
+        <ActionTypeRow actionType={action.type} />
+        <ActionData
+          rowLabel={formatText({ id: 'actionSidebar.member' })}
+          tooltipContent={formatText({
+            id: 'actionSidebar.tooltip.manageReputation.member',
+          })}
+          RowIcon={UserFocus}
+          rowContent={
+            <UserAvatar
```

Comment:
cc @ArmandoGraterol do we need support for just wallets as recipients too?

Reply:
By just wallets, you mean users without profiles no? If that's the case, yes, we should always support displaying users without profiles when it comes to actions due to the forms making it possible to insert ANY wallet address. Users with no profile may not be able to access colonies now due to the private beta but that will change in the future.